### PR TITLE
WL-1403 | suppressing digest notification for WL-sites.

### PR DESCRIPTION
**Pull request by hasnain-naveed**

### CODE REVIEW

**Comment by bill-filler** [comment]
@iloveagent57 
Would you mind giving this a quick look? Would be a short term fix to suppress digest emails for Wharton

File: notifier/tasks.py [hunk 1]
```diff
@@ -36,13 +36,26 @@ def generate_and_send_digests(users, from_dt, to_dt, language=None):
...
     msgs = []
+
+    for user in users:
+        skip_user = False
+        for course_id in user["course_info"].keys():
+            for org in orgs_to_skip:
+                if org in course_id:
+                    skip_user = True
+        if skip_user:
+            del users_by_id[str(user["id"])]
```
Comment (iloveagent57) [inline notifier/tasks.py]: If we do this, the user won't get a digest for _any_ course, which isn't what we want - we just don't want to send digests for Wharton courses.  I think the right place to do the filtering is in `generate_digest_content`.  Let me take a look at that for a bit and get back to you.

**Comment by iloveagent57** [comment]
I think the easiest place to make this change will be here: https://github.com/edx/notifier/blob/master/notifier/pull.py#L79

You could do the following:
* Write a little helper function that checks if a course_id contains any string corresponding to an org we don't want to send digests for (you already have that logic in this PR).  Call it something like `_should_skip_org(course_id)`.
* Use that function at the beginning of the `_build_digest_course()` function.  If `_should_skip_org(course_id)` returns true, return an empty `DigestCourse` object - you can create one by doing `DigestCourse(course_id, threads=[])`.
* This should cause `_build_digest()` to skip over the course digest for any course with an org that we don't want to send digests for.

**Comment by iloveagent57** [comment]
There are some docs here that should help you when testing this on a sandbox: https://openedx.atlassian.net/wiki/spaces/EDUCATOR/pages/46794261/Notifier

**Comment by bill-filler** [comment]
@iloveagent57 
thanks for the tips!
@hasnain-naveed 
please make the changes Alex suggested and resubmit

**Pull request merged**